examples/next-prisma-websockets-starter/src/pages/api/auth/[...nextauth].ts

```diff
@@ -20,10 +20,12 @@ if (useMockProvider) {
       name: 'Mocked GitHub',
       async authorize(credentials) {
         if (credentials) {
+          const name =
+            typeof credentials.name === 'string' ? credentials.name : 'test';
           const user = {
-            id: credentials.name,
-            name: credentials.name,
-            email: credentials.name,
+            id: name,
+            name: name,
+            email: name,
```

Comment:
the next-auth@5 beta im using to get a nice story for the app router seems to have changed the type here, hence the added check. could also `as any` it i guess